doc fixes

cota opened the pull request:
Hi Samy,

I've started using ck_hs/ht and when looking at the man pages I found a couple of errors in them. In this set I'm fixing these errors and also adding checks to prevent them from happening in the future.

BTW when building the HTML pages, `man2html` was always getting the `cgiurl=$$title` flag, and $$title wasn't defined anywhere. Was that intentional? I assumed it wasn't, and changed it to `cgiurl=$$target`.

sbahra commented:
Hi Emilio,

Thanks for the patches. No, it wasn't intentional. I've also been running into a bug with man2html that causes is to generate broken pages.

cota commented:
I've rebased the branch (just noticed this system doesn't keep track of the rebase, which is confusing), removing the first commit that didn't work with BSD make (so no parallel builds). I've also fixed the objcheck commit to make it work with BSD make (it doesn't seem to like $(shell foo), used `foo` instead).

sbahra commented:
Perfect, thank you! This is very useful.
